src/mcode-pilot/services/outcomes/CLQOutcomesService.js

```diff
@@ -1,35 +1,22 @@
 import request from "request";
-import _ from 'lodash';
 import IOutcomesService from './IOutcomesService';
+import FilterOptions from '../../utils/FilterOptions';
+
 export default class CLQOutcomesService extends IOutcomesService {
     constructor(params) {
         super();
         this.serviceUrl = params.serviceUrl;
         this.timescale = params.timescale || [];
     }
 
-    /*
-    Based off of the mcode namespace find the elements that match it in the filter
-    */
-    findMcodeElements(props, mcodeType, single = false) {
-        let found = [];
-        _.forIn(props, function (value, key) {
-            if (value.mcodeElement === mcodeType && value.selected) {
-                found.push(value);
-            }
-        });
-        return single ? found[0] : found;
-    }
-
     /* Build the CLQ demograpchics filter section based off of the Compass filter criteria
      */
-    buildDemographicsFilter(similarPatientProps) {
-        let demoProps = similarPatientProps.demographic.options;
+    buildDemographicsFilter(filterOptions) {
```

Comment:
This variable is called `filterOptions` but it's not actually an instance of the `FilterOptions` class, it's the result of `FilterOptions.getAllActiveValuesByMcodeElement()`. Can you make that more clear? Or possibly just add some JSDoc to clarify what this is.  Same comment for `buildDiagnosisFilter` and `buildTumorMarkersFilter`